include/C#/SAMP_API.cs

```diff
@@ -121,19 +143,44 @@ class API
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
         public static extern float GetVehicleHealth();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern short GetVehicleTypeId();
+        public static extern int GetVehicleModelId();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int IsVehicleBike();
-
+        public static extern int GetVehicleModelName(ref StringBuilder name, int len);
+        public static extern int GetVehicleModelNameById(int vehicleId, ref StringBuilder name, int len);
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int GetPlayerIDByName(string name);
+        public static extern int GetVehicleType();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int GetPlayerNameByID(int id, ref StringBuilder playerName, int maxLen);
-
+        public static extern int GetVehicleFreeSeats(ref int seatFL, ref int seatFR, ref int seatRL, ref int seatRR);
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int Init();
+        public static extern int GetVehicleFirstColor();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern void SetParam(string _szParamName, string _szParamValue);
+        public static extern int GetVehicleSecondColor();
+        [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
+        public static extern int GetVehicleColor(ref int color1, ref int color2);
```

Comment:
I wonder if we should streamline `ref` and `out`. http://stackoverflow.com/a/1516890/3455922 suggests `out` is better for multiple reasons. Any thoughts on this, @shadowlif?